install/hst-install-ubuntu.sh

```diff
@@ -1784,6 +1800,17 @@ if [ "$mysql" == 'yes' ] && [ "$dovecot" == "yes" ]; then
     write_config_value "WEBMAIL_ALIAS" "webmail"
 fi
 
+#----------------------------------------------------------#
+#                     Configure Sieve                      #
+#----------------------------------------------------------#
+# Min requirements Dovecote + Exim + Mysql + roundcube
+if [ "$sieve" = 'yes' ]; then
+    echo "[ * ] Configuring Sieve ..."
+    $HESTIA/bin/v-add-sys-sieve
+else
+    # This should uninstall sieve if you had installed before
+    $HESTIA/bin/v-add-sys-sieve
```

Comment:
Installer is not build for running again  so it is better to remove it...

Reply:
> nstaller is not build for running again so it is better to remove it...

I remove that from hst-install-ubuntu.sh ;-)
Best regards!